sdk/formrecognizer/Azure.AI.FormRecognizer/tests/ReceiptClient/ReceiptClientLiveTests.cs

```diff
@@ -43,16 +49,16 @@ public ReceiptClientLiveTests(bool isAsync) : base(isAsync)
         ///
         [Test]
         [TestCase(true)]
-        [TestCase(false)]
-        [Ignore("The receipt file has not been uploaded yet.")]
+        [TestCase(false, Ignore="The receipt file has not been uploaded to GitHub yet.")]
```

Comment:
Yes. I could use the URI path from the [Azure Samples](https://raw.githubusercontent.com/Azure-Samples/cognitive-services-REST-api-samples/master/curl/form-recognizer/contoso-receipt.png) in GitHub and successfully run the test. We need to ignore it for now since the file is not in the repo yet (it will be once this PR is merged).